app/views/content/shared/age-groups-and-specialisms/_teaching_send.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="teaching_send">
+  <h3>Pupils with special educational needs and disabilities (SEND)</h3>
+  <p>All initial teacher training courses cover teaching pupils with SEND. Whether you train to teach in mainstream schools, special schools, or both, you’ll <a href="/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs">teach pupils with SEND</a>, and learn how to support their needs.</p>
```

Comment:
```suggestion
  <p>All initial teacher training courses cover teaching pupils with SEND. Whether you train to teach in mainstream schools, special schools, or both, you’ll <a href="/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs">teach pupils with special educational needs and disabilities (SEND)</a>, and learn how to support their needs.</p>
```
Should we replace/add to  the acronym in the link text? I'm thinking of screen reader users who might be skipping through links and not necessarily reading the h3 first, so they may not understand the acronym? One to discuss with @Sarah-DfE